frontend/src/pages/videos/VideoAnalysis.tsx

```diff
@@ -8,10 +8,12 @@ import VideoCard from 'src/features/videos/VideoCard';
 import { useVideoMetadata } from 'src/features/videos/VideoApi';
 import CriteriaBarChart from 'src/components/CriteriaBarChart';
 import { VideoPlayer } from 'src/components/entity/EntityImagery';
+import CriteriaScoreDistributions from 'src/components/CriteriaScoreDistributions';
 
 function VideoAnalysis() {
   const { video_id } = useParams<{ video_id: string }>();
   const video = useVideoMetadata(video_id);
+  const entityId = `yt:${video_id}`;
```

Comment:
I'm not sure how I should get this, but I guess in the future this page will probably become an `EntityAnalysis` and have a real entity id so this won't be needed.